### DISCUSSION TITLE
Using Zotero to sort the finished screening after labels

### DISCUSSION THREAD

**Discussion by pallykkja** [asreview/asreview#1619] [Q&A] [answered]
Hello,
I have tried to sort RIS files so that I could group them in Zotero. I would like to hear if it is a better method than this. I have put it into a Powerpoint presentation. See the file.
[How to sort ASReview with Zotero.pptx](https://github.com/asreview/asreview-academy/files/13751872/How.to.sort.ASReview.with.Zotero.pptx)
Best, Pål Magnus Lykkja p.m.lykkja@ub.uio.no

**Comment by J535D165** [comment]
Did you try the "All fields & tags" search field (top right)? I search for ASReview_relevant and then drag them to a newly created subcollection. This works without the use of extensions.

**Reply by pallykkja** [reply, marked as answer]
Thank you very much! It didnt work for me when I ctrl + A, and tried to drag and drop. But, when I held down Shift, marked all I wanted, then I could drag and drop it. It should be mentioned in the ASReview how to section somewhere, because it is very useful to sort out relevant, irrelvant and not seen in different sections.

**Reply by J535D165** [reply]
I faced the same when I tried. Good to see you figured out. 

@LHofstee might be relevant for the workshops/academy material.

**Comment by pallykkja** [comment]
[Dedublisering i Zotero.pptx](https://github.com/asreview/asreview/files/13802061/Dedublisering.i.Zotero.pptx)
Just in case it is interesting, this method with confirming DOI makes the deduplication surprising fast I would say. At leas when one are used to the operations in Zotero

**Reply by J535D165** [reply]
Thanks for sharing! Relevant for our team! 

cc @laurens88

**Reply by pallykkja** [reply]
Glad to hear that. With this mix of deduplication automatically only the verified doi journal articles, then I could use the great manual deduplication method to do the harder duplication issues. In the dataset it was 1792 unique items, and with my method, I got 1794. It could be that the specificity and sensitity is low, but that would be easy to check with the data. I think it may be the fastest and most precice method for deduplication. The only thing is that with Zotero, one has to just let the software work even if it looks like nothing happen, or it looks like freezing. When you restart or continue, it has (almost) always been correct as far as I see. The use of the Garbidge bin is so useful to isolate deduplication processes on verified doi journal articles, but it has to be only journal articles, because then you cant remove all the 1200+ duplicates in two click with the mouse. Zotero dont deduplicate different item types. But since most is journal articles, it save a lot of time and precision.

**Reply by pallykkja** [reply]
I did a big mistake, it is not possible to mark all and merge, you have to do it one by one, or use the https://github.com/frangoud/ZoteroDuplicatesMerger ( I think it is good to sort by Item Type in Zotero, put everything else except journal articles in the garbage bin, otherwise it will stop frequently), and then restore all the sources to library and do it manually the rest.
